charts/microgateway/README.md.gotmpl

```diff
@@ -226,431 +231,372 @@ In case that the [Simple DSL configuration](#simple-dsl-configuration) does not
-In the following chapters, various examples of these configuration options are given to help you better understand the DSL mechanism in order to use it as efficiently as possible.  
-Depending on the requirements of the Microgateway configuration, a different DSL must be created.   
-
-#### Simple setup example 
-The Simple Setup assumes that it has a virtual host, mapping and backend each.   
-For the Simple Setup all settings `config.*` are available.  
-Thus, the user does not have to write and provide his own DSL. The generation of the DSL with the required values is done by the Helm chart.  
-
-simple-values.yaml
-```
-config:
-  simple:
-    mapping:
-      operationalMode: integration
-      denyRules:
-        level: strict
-    backend:
-      hostname: backend-hostname
-  global:
-    logLevel: trace
-```
-
-#### Advanced setup example
-The Advanced setup should be used if the settings of the Simple Setup are no longer sufficient.  
-Therefore, if more than one virtual host, mapping and backend is needed, or if the app requires different settings than those that can be set with the `config.default.*`.   
-Please note that a valid DSL app must be specified and the values config.default.* are no longer used.   
-
-The following two examples show a possible use case where you could choose the Advanced Setup.  
-
-Multiple Mappings:
-```
-apps:
-  - backend:
-      hostname: backend-hostname
-      protocol: https
-    mappings:
-      - name: root
-      - name: example
-        entry_path: /example/
-        operational_mode: integration
-```
-
-Deny Rule exceptions:
-```
-apps:
-  - backend:
-      hostname: backend-hostname
-      protocol: https
-    mappings:
-      - name: root
-        deny_rules:
-          - level: strict
-            exceptions:
-              - parameter_name:
-                  pattern: $exception.*parameter^
-                parameter_value:
-                  pattern: $exception.*value^  
-```
-
-#### Expert setup example
-In the expert setup you must dispense of all config.* settings of the Helm chart, with the exception of the config.dsl setting.  
-This means that if you have DSL requirements that cannot be covered by the first two setups, you can specify your own DSL.   
-
-### External connectivity
-This section describes how the external connectivity can be configured.   
-There are two different scenarios if you are on Kubernetes or if you want to install on Openshift.   
-On Kubernetes an Ingress Controller is used and on Openshift a Route object.   
+The Helm chart is already pre-configured for the readiness probe endpoint of the Microgateway Pod. A huge Microgateway configuration could require to increase the initial delay. 
+This can be accomplished by configuring the following parameter:
+
+  ```
+  readinessProbe:
+    initialDelaySeconds: 90
+  ```
+
+If desired, the readiness probe can be disabled with `readinessProbe.enabled=false`.
```

Comment:
If you disable the ReadinessProbe from our chart the default of the environment is used.

Reply:
Done